src/AtmosphereModels/atmosphere_model.jl

```diff
@@ -100,21 +113,20 @@ function AtmosphereModel(grid;
                          coriolis = nothing,
                          boundary_conditions = NamedTuple(),
                          forcing = NamedTuple(),
-                         advection = nothing,
-                         momentum_advection = nothing,
-                         scalar_advection = nothing,
+                         advection = DefaultValue(),
+                         momentum_advection = DefaultValue(),
+                         scalar_advection = DefaultValue(),
                          closure = nothing,
                          microphysics = nothing, # WarmPhaseSaturationAdjustment(),
                          timestepper = :RungeKutta3,
-                         radiation = nothing,
```

Comment:
Radiation was used in https://github.com/NumericalEarth/Breeze.jl/blob/2570353a3c96bdbb1f49a0090c26e708ecf2673f/test/radiative_transfer.jl

Reply:
Makes sense, given that one was being set equal to the other.